node-gyp error - binding.gyp not found

Attempting to install the latest version (2.12.0) of dd-trace is giving the following error: 
`npm ERR! gyp: binding.gyp not found (cwd: #####/node_modules/@datadog/native-metrics) while trying to load binding.gyp
npm ERR! gyp ERR! configure error`

My system info is:
Node: 16.13.1
OS: macOS Big Sur 11.6.8

I've attempted:
- Deleting node_modules and package-lock file
- Re-installing node-gyp
- Re-installing node itself
- Installing an older version of dd-trace: 1.5.1

If I remove dd-trace from my package.json file, I can install other packages, etc. successfully. 

Any help on this issue would be much appreciated. Thank you!

--- a/package.json
+++ b/package.json
@@ -59,8 +59,8 @@
   },
   "dependencies": {
     "@datadog/native-appsec": "^1.2.1",
-    "@datadog/native-metrics": "^1.4.1",
-    "@datadog/pprof": "^1.0.0",
+    "@datadog/native-metrics": "^1.4.2",
+    "@datadog/pprof": "^1.0.2",
     "@datadog/sketches-js": "^1.0.5",
     "@types/node": ">=12",
     "crypto-randomuuid": "^1.0.0",

